sensor/common/sensor/env.go

```diff
@@ -0,0 +1,39 @@
+package sensor
+
+import (
+	"time"
+
+	"github.com/stackrox/rox/pkg/env"
+)
+
+const (
+	defaultInitialInterval = 10 * time.Minute
+	defaultMaxInterval     = time.Minute
+)
+
+var (
+	connectionRetryInitialIntervalEnv = env.RegisterSetting("ROX_SENSOR_CONNECTION_RETRY_INITIAL_INTERVAL")
```

Comment:
There is a [DurationSetting](https://github.com/stackrox/stackrox/blob/13e9ae510df517706b64b848bca4aa78a8b9d6c0/pkg/env/durationsetting.go#L17) in the `env` pkg. I'm not sure if there's a reason to why the `registerDurationSetting` is not exported but I wonder if it would be make sense to export it and use it here.

Reply:
Nice. I moved the envs to the `pkg/env` package since we already had a few sensor envs there.